src/eureka/S6_planet_spectra/s6_spectra.py

```diff
@@ -722,7 +726,7 @@ def load_s5_saves(meta, log, fit_methods, n_samples=1):
         A list of sample arrays, one for each channel.
     """
     if 'dynesty' in fit_methods:
-        if meta.run_dynamic:
+        if hasattr(meta, 'run_dynamic') and meta.run_dynamic:
```

Comment:
`run_dynamic` should be defined in s6_meta.py so that we don't have to check that is exists using `hasattr`.